todo/src/App.js

```diff
@@ -18,19 +21,24 @@ function App() {
     event.preventDefault();
     setListOfTodos([
       ...listOfTodos,
+      // What's a nanoid? What are you using it for?
       { text: todo, edit: false, id: nanoid(), sort: count },
     ]);
     setCount(count + 1);
     setTodo("");
   };
 
   const handleDelete = (event) => {
+    // Does this mean if you have two of the same todo and delete one, both will be deleted?
+    // Is this what you expect?
+    // I would have assumed the IDs were to avoid this?
     let filteredList = listOfTodos.filter((todo) => todo.text !== event.text);
     setListOfTodos(filteredList);
     setTodo("");
   };
 
   const handleEdit = (event) => {
+    // Same as delete - does this mean editing one will edit the first matching one, even if that's not the one you clicked on?
     let currentTodo = listOfTodos.find((todo) => todo.text === event.text);
```

Comment:
@illicitonion  No this one is different, inside of handleEdit I have if and else for changing the setEditAvailable but i feel like I can change the code inside of this function 🤔. I will think about it.